src/features/advanced/exp-expert-eta.ts

```
@@ -0,0 +1,168 @@
+  'FUEL_REFINING',
+  'MANUFACTURING',
+  'METALLURGY',
+  'RESOURCE_EXTRACTION',
+];
+
+// https://handbook.apex.prosperousuniverse.com/wiki/efficiency-factors/index.html#expert-spawn-rates--bonuses
+const expertDays = [10.0, 12.5, 57.57, 276.5, 915.1];
+const MS_IN_DAY = 24 * 60 * 60 * 1000;
+
+function onTileReady(tile: PrunTile) {
+  const site = sitesStore.getById(tile.parameter)!;
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)